File: test_03/src/main/java/EntryPoint.java
/**
 * This is entry point class
 */
public class EntryPoint {
  public static void main(String[] args) {
    ValidatorOfTheAlgebraicExpression validatorOfTheAlgebraicExpression = new ValidatorOfTheAlgebraicExpression("{a}");
    if (validatorOfTheAlgebraicExpression.makeCheck()) {
      System.out.println("Valid expression");
    } else {
      System.out.println("Not a valid expression");
    }
  }
}

File: AlgebraicExpression/EntryPoint.cs
using System;

namespace AlgebraicExpression
{
  /// <summary>
  /// Entry point to program.
  /// </summary>
  class EntryPoint
  {
    static void Main(string[] args)
    {
      try
      {
        Expression expression = new Expression(Console.ReadLine());
        BracketsChecker bracketsChecker = new BracketsChecker(expression.GetBrackets());
        Console.WriteLine(bracketsChecker.CheckBrackets());
      }
      catch(Exception ex)
      {
        Console.WriteLine(ex.Message);
      }
    }
  }
}